Makefile

```diff
@@ -33,7 +33,7 @@ endif
 all: $(OUT_DIR)/$(ARCH)/adapter
 
 src_deps=$(shell find pkg cmd -type f -name "*.go")
-$(OUT_DIR)/%/adapter: $(src_deps)
+$(OUT_DIR)/%/adapter: pkg/api/generated/openapi/zz_generated.openapi.go $(src_deps)
```

Comment:
this file should already be part of the `src_deps` variable

Reply:
Done